db-service/lib/infer/index.js

```diff
@@ -798,7 +798,7 @@ function infer(originalQuery, model = cds.context?.model || cds.model) {
     function resolveCalculatedElement(calcElement) {
       if (alreadySeenCalcElements.has(calcElement)) return
       else alreadySeenCalcElements.add(calcElement)
-      const { ref, val, xpr, func } = calcElement.value
+      const { ref, /* val, */ xpr, func } = calcElement.value
```

Comment:
@patricebender is this fine ?

Reply:
👍
```suggestion
      const { ref, xpr, func } = calcElement.value
```

Reply:
i just wanted to get rid of the warning, but didn't want to delete stuff from coding i'm not familiar with ;)